store/piecestore/storage/b2.go

```diff
@@ -98,6 +98,6 @@ func parseB2BucketURL(bucketURL string) (endpoint, region, bucketName string, er
 		isVirtualHostStyle = true
 	}
 
-	region = parseRegion(endpoint)
+	region = parseS3Region(endpoint)
```

Comment:
Why need to change B2?

Reply:
Because B2 uses s3 `parseRegion` funciton